File: JS/Richest Customer Wealth.js
/**
 * @param {number[][]} accounts
 * @return {number}
 */ 
var maximumWealth = function(accounts) {
    let maximum=0;
    for(let i=0;i<accounts.length;++i){
        let sum=0;
        for(let j=0;j<accounts[i].length;++j){
            sum += accounts[i][j];
        }
       maximum=Math.max(sum,maximum); 
    }
   return maximum; 
};

File: C++/Richest Customer Wealth.cpp
class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {
        int maximum=0;
        for(auto acc : accounts){
            int sum=0;
            for(auto i : acc){
                sum += i;
            }
           maximum=max(maximum,sum); 
        }
       return maximum; 
    }
};